core/cx_pool.go

```diff
@@ -0,0 +1,55 @@
+package core
+
+import (
+	mapset "github.com/deckarep/golang-set"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	// CxPoolSize is the maximum size of the pool
+	CxPoolSize = 50
+)
+
+// CxEntry represents the egress receipt's blockHash and ToShardID
+type CxEntry struct {
+	BlockHash common.Hash
+	ToShardID uint32
+}
+
+// CxPool is to hold a pool of block outgoing receipts to be resend in next round broadcast
+type CxPool struct {
+	pool    mapset.Set
```

Comment:
Add comment for what's the expected data to be stored in this set.

Reply:
added